src/opnsense/mvc/app/controllers/OPNsense/Interfaces/forms/dialogVxlan.xml

```diff
@@ -20,6 +20,13 @@
           When the interface is configured in unicast mode, the listening socket is bound to this address.
         </help>
     </field>
+    <field>
+        <id>vxlan.vxlanlocalport</id>
+        <label>Source port</label>
+        <type>text</type>
+        <help>Define the source port. Default is 4789.</help>
```

Comment:
If the goal is to default to that value automatically I don’t think it works. We should not break prior setups so leaving empty must be allowed or migrated, but I see no reason to migrate and force people to enter a default port from now on especially when hidden under advance d.

Reply:
Thanks for the feedback. I will change the approach to it.

Reply:
Thanks. Don’t mind me nagging. You are doing good 👍

Reply:
Thank you very much. I don't mind it at all, because you are right about these things. I'm very open minded in general, and I always think that what I try can be improved.  :+1: